AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -52,49 +57,88 @@ private extension SearchMapView {
-                
+        ZStack(alignment: .top) {
+            SearchAndFollowMap(startingPoint: viewModel.passedLocationAddress,
+                               showSearchAgainButton: $viewModel.searchAgainButton,
+                               sessions: $viewModel.sessionsList,
+                               pointerID: $viewModel.cardPointerID).onPositionChange(action: { geoSquare in
+                viewModel.mapPositionsChanged(geoSquare: geoSquare)
+            })
+                               .padding(.top, 50)
+                               .ignoresSafeArea(.all, edges: [.bottom])
+            LinearGradient(gradient: Gradient(colors: [.white.opacity(0.1),
+                                                       .white.opacity(0.5),
+                                                       .white.opacity(0.7),
+                                                       .white.opacity(0.8),
+                                                       .white.opacity(0.9),
+                                                       .white]),
+                           startPoint: .bottom,
+                           endPoint: .top)
+            .frame(width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height / 4.5, alignment: .top)
+        }
+    }
+    
+    var cardsTitle: some View {
+        HStack(alignment: .bottom) {
+            StringCustomizer.customizeString(String(format: Strings.SearchMapView.cardsTitle,
+                                                    arguments: ["\(viewModel.sessionsList.count)",
+                                                                "\(viewModel.sessionsList.count)"]),
+                                             using: [Strings.SearchMapView.sessionsText],
+                                             color: .darkBlue,
+                                             standardColor: .darkBlue,
+                                             font: Fonts.boldHeading2)
+        }.foregroundColor(.darkBlue)
+    }
+    
+    var cards: some View {
+        ScrollViewReader { scrollProxy in
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(alignment: .bottom, spacing: 12) {
+                    ForEach(viewModel.sessionsList, id: \.id) { session in
+                        BottomCardView(cardPointer: $viewModel.cardPointerID,
+                                       dataModel: .init(id: session.id,
+                                                        title: session.title,
+                                                        startTime: session.startTime,
+                                                        endTime: session.endTime,
+                                                        latitude: session.location.latitude,
+                                                        longitude: session.location.longitude))
+                        .border((viewModel.cardPointerID == session.id ? .blue : .clear), width: 1)
```

Comment:
Can we give the border an aircasting blue color?

Reply:
Suree 🥳